net/pkt/pkt_sendmsg.c

```diff
@@ -113,7 +113,7 @@ static uint16_t psock_send_eventhandler(FAR struct net_driver_s *dev,
               goto end_wait;
             }
 
-          dev->d_len       = dev->d_sndlen -= NET_LL_HDRLEN(dev);
+          dev->d_len       = dev->d_sndlen;
```

Comment:
I had question to exactly this line in https://github.com/apache/nuttx/pull/12688/files#r1683507533

Reply:
This length (dev->d_len) is to ensure that the data length sent by the network card driver in the lower half is valid. Do you have any suggestions for changes here?